pkgs/development/compilers/ghc/8.10.4.nix

```diff
@@ -91,7 +91,8 @@ let
 
   # ld.gold is disabled for musl libc due to https://sourceware.org/bugzilla/show_bug.cgi?id=23856
   # see #84670 and #49071 for more background.
```

Comment:
```suggestion
  # Use gold either following the default, or to avoid the BFD linker due to some bugs / perf issues.
  # But we cannot avoid BFD when using musl libc due to https://sourceware.org/bugzilla/show_bug.cgi?id=23856
  # see #84670 and #49071 for more background.
```

Reply:
Done!